velox/common/base/BitUtil.h

```diff
@@ -118,6 +118,18 @@ inline int32_t getAndClearLastSetBit(uint16_t& bits) {
   return trailingZeros;
 }
 
+#if __ARM_ARCH
+inline int32_t arm64_getAndClearLastSetBit(uint64_t& bits) {
+  int32_t trailingZeros = __builtin_ctzll(bits) >> 2;
+  // erase last 4 non-zero bit
+  bits &= bits - 1;
```

Comment:
I don't think it is correct to do this 4 times.  Also rename it to `getAndClearLast4SetBit` and remove `#if __ARM_ARCH` check